extension/lib/doc.js

```diff
@@ -181,45 +190,54 @@ class Doc {
                     termFreq[word]++;
                 } else {
                     termFreq[word] = 1;
-                }
+                };
```

Comment:
remove this semicolon?

Reply:
Good catch:
(Good resource w/ the rules; didn't know some of the cases, will try to remove others as I see them)
https://news.codecademy.com/your-guide-to-semicolons-in-javascript/